### PR TITLE
Remove fluentui Link and Text components and replace with MUI equivalents

### PR DESCRIPTION
**Pull request by 2metres**
**User-Facing Changes**

**Description**

### CODE REVIEW

**Comment by defunctzombie** [comment]
@2metres is this PR ready for review or is it still a draft?

**Comment by 2metres** [comment]
@defunctzombie this is ready for review

File: packages/studio-base/src/components/AccountSettingsSidebar/AccountInfo.tsx [hunk 1]
```diff
@@ -53,34 +46,31 @@ export default function AccountInfo(props: { currentUser?: User }): JSX.Element
...
+        <Stack direction="row" alignItems="center" gap={1}>
           <Icon
             iconName="BlockheadFilled"
             styles={{
               root: {
-                color: theme.palette.themePrimary,
-                fontSize: theme.fonts.superLarge.fontSize,
-                height: theme.fonts.superLarge.fontSize,
+                color: theme.palette.primary.main,
+                fontSize: 42,
```
Comment (defunctzombie) [inline packages/studio-base/src/components/AccountSettingsSidebar/AccountInfo.tsx:57]: Should we be hard coding this? Or should we be using theme sizing?

File: packages/studio-base/src/theme/palette.ts [hunk 2]
```diff
@@ -19,7 +19,7 @@ export const dark: PaletteOptions & CustomPaletteOptions = {
   info: { main: "#29bee7" },
   text: {
     primary: "#e1e1e4",
-    secondary: "#bbbac0",
+    secondary: "#a7a6af",
```
Comment (defunctzombie) [inline packages/studio-base/src/theme/palette.ts:22]: Why this tweak in this PR? Did we have the colors wrong?

**Review by defunctzombie — APPROVED** [review]

File: packages/studio-base/src/theme/palette.ts (hunk 2 above)
Comment (2metres) [reply packages/studio-base/src/theme/palette.ts:22]: The fluent colors and mui colors were different. I think IMO the fluent secondary text color was better.

File: packages/studio-base/src/components/AccountSettingsSidebar/AccountInfo.tsx (hunk 1 above)
Comment (2metres) [reply packages/studio-base/src/components/AccountSettingsSidebar/AccountInfo.tsx]: At the end of the day the number is going to come from somewhere, right. I've moved it into a `const` given it was a bit arbitrary.

**Pull request merged**